src/public/javascripts/signUpAgree.js

```diff
@@ -1,12 +1,86 @@
-const $agreeAll = document.querySelector('.agree-box-wrapper');
-const $agreeAllBtn = document.querySelector('.agree-toggle-btn');
+import { goSignUpPhonePage } from './utils/router.js';
 
-function toggleAgree($boxImg) {}
+const $agreeBtnList = document.querySelectorAll('.agree-toggle-btn');
 
-$agreeAllBtn.addEventListener('click', (e) => {
-  console.log('hit');
-  $agreeAll.classList.toggle('option');
+const $ageOldBtn = document.querySelector('.age-old-btn');
+const $ageYoungBtn = document.querySelector('.age-young-btn');
 
-  const $boxImg = $agreeAllBtn.querySelector('.box-img');
-  toggleAgree($boxImg);
+const $nextBtn = document.querySelector('.square-button');
+
+$agreeBtnList.forEach(($agreeBtn) => {
+  $agreeBtn.addEventListener('click', (e) => {
+    if ($agreeBtn.classList.contains('all')) {
+      toggleBtnAll($agreeBtn, $agreeBtnList);
+      return;
+    } else if ($agreeBtn.classList.contains('normal')) {
+    }
+
+    toggleBtn($agreeBtn);
+  });
```

Comment:
```suggestion
  $agreeBtn.addEventListener('click', (e) => {
    if ($agreeBtn.classList.contains('all')) {
      toggleBtnAll($agreeBtn, $agreeBtnList);
    } else {
      toggleBtn($agreeBtn);
    }
  });
```
if 문에서 return을 하고, else if 문엔 코드가 없고, 그 밑엔 toggleBtn 메서드가 있으니 위 코드와 같은걸까요?

Reply:
`$agreeBtnList.forEach(($agreeBtn) => {
  $agreeBtn.addEventListener('click', (e) => {
    if ($agreeBtn.classList.contains('all')) {
      toggleBtnAll($agreeBtn, $agreeBtnList);
    } else {
      toggleBtn($agreeBtn);
    }

    if (checkAgree($agreeBtnList)) {
      $nextBtn.classList.remove('disabled');
    } else {
      $nextBtn.classList.add('disabled');
    }
  });
});`

이렇게 변경했습니다.